styles/style.css

```diff
@@ -1,10 +1,217 @@
+/*inserted by Prabesh ------------------------------------------------------------- 
+*Bikes-for-refugees
+
+*/
+
+/* temp styling */
+
+
+
+/* main styling for code your futrue week 1 tutorial by PRABESH recreation of the page without using bootstrap*/
+header {
+    max-width:85%;
+    margin:0 auto;
+    display:flex;
+    flex-wrap:wrap;
+}
+.logo-img {
+    height:50px;
+    width:158px;
+}
+.navigation-ul {
+    list-style:none; 
+}
+.navigation-ul li {
+    float:left;
+    width:10%;
+}
+.navigation-ul li:nth-child(3), .navigation-ul li:nth-child(4) {
```

Comment:
This is difficult to read...how about adding a class to the two navigation elements you are targeting?